python3/libexec/nbd_client_manager.py

```diff
@@ -206,7 +212,17 @@ def connect_nbd(path, exportname):
                     "-name",
                     exportname,
                 ]
-                _call(cmd)
+                ret = _call(cmd, raise_err=False, log_err=True)
+                if NLE_BUSY == ret:
+                    # Although _find_unused_nbd_device tell us the nbd devcie is
+                    # not connected by other nbd-client, it may be opened and locked
+                    # by other process like systemd-udev, raise NbdDeviceNotFound to retry
```

Comment:
Another way is to catch the exception `subprocess.CalledProcessError` and check the return code in it: if it is NLE_BUSY from the expected cmd, continue, otherwise, re-raise the execption.

Reply:
that way would harder to share `NbdDeviceNotFound` exception. 
BTW, the `NbdDeviceNotFound` is actually also device busy actually.